### ISSUE TITLE
Full Text returns wrong results for Turkish

### ISSUE THREAD

**Issue by ogun** [bayard-search/bayard#2]
It does not provide a correct search result when the word contains "Turkish i" char.

Index documents
```
./bin/bayard set 1 '{ "text" : "quıt" }'
./bin/bayard set 2 '{ "text" : "quit" }'
./bin/bayard set 3 '{ "text" : "QUIT" }'
./bin/bayard set 4 '{ "text" : "QUİT" }'
```

Search for "quit":
```
./bin/bayard search text:"quit"
```

Actual Results
```
[
  {"id":["3"],"text":["QUIT"]},
  {"id":["2"],"text":["quit"]}
]
```

Expected Results
```
[
  {"id":["4"],"text":["QUİT"]},
  {"id":["3"],"text":["QUIT"]},
  {"id":["2"],"text":["quit"]},
  {"id":["1"],"text":["quıt"]}
]
```

**Comment by mosuka** [comment]
@ogun 
Thank you for your report.
The example schema (https://github.com/bayard-search/bayard/blob/master/etc/schema.json) is very simple.
Since Bayard built on top of Tantivy, the support status of each language depends on whether Tantivy supports it.
Please contact the Tantivy project. https://github.com/tantivy-search/tantivy
Thanks,

**Comment by mosuka** [comment]
Hi @ogun ,
I released Bayard 0.4.0. It supports some language specific pre-configured tokenizers.
You can index and search Turkish documents as follows:

```
$ ./bin/bayard serve --schema-file=./etc/schema_tr.json
```

```
$ ./bin/bayard schema
```

```
$ ./bin/bayard put --id=1 --file=./examples/doc_tr.json
```

```
$ ./bin/bayard search description:Güneydoğu
$ ./bin/bayard search description:Guneydogu
```
